### ISSUE TITLE
Latest shoutcast not working with Azuracast

### ISSUE THREAD

**Issue by vpsinghbaghel** [AzuraCast/AzuraCast#4821]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### What happened?

Freshly installed on Ubuntu 20.
All works fine before ShoutCast installation.

Installing latest version of ShoutCast and then shoutcast doesn't work at all.

I have tried the same on different VPS. Same issue with shoutcast.

Does shoutcast not support Ubuntu 20 ?

Please help.

### Relevant log output

```shell
Getting this error when starting Shoutcast broadcast.

App\Exception\Supervisor\BadNameException at /var/azuracast/www/src/Radio/AbstractAdapter.php L233: SHOUTcast is not recognized as a service.
It may not be registered with Supervisor yet. Restarting broadcasting may help.
```

**Comment by SpeedyCharly** [comment]
I am currently running Shoutcast Server v2.6.0.753 on a Ubuntu 20 machine with no problem whatsoever...

**Comment by vpsinghbaghel** [comment]
Thank you very much for responding.

It was working absolutely fine for me too until few days ago with last install.
I am setting up Azuracast on a new Ubuntu20 VPS. Azuracast works fine with icecast.
When I install Shoutcast, it installs successfully. Not not started. Getting above error when trying to start it.
Shoutcast service starts when restarting station. But still station shows offline. It doesn't work at all.

If you talk about any already installed setup, it doesn't prove the issue I am mentioning.
Please setup new and if it works fine then it will mean I am doing something wrong or I have another issue with my server.

Thank you again.
Best Regards.
Ved.

**Comment by SpeedyCharly** [comment]
Is it possible that the ShoutCast installation you're attempting to run requires licence information?

**Comment by vpsinghbaghel** [comment]
> Is it possible that the ShoutCast installation you're attempting to run requires licence information?

Maybe. I am not sure.
How to get free version of shoutcast?
I have downloaded shoutcast from a paid account.

**Comment by SpeedyCharly** [comment]
Info;
http://wiki.shoutcast.com/wiki/SHOUTcast_DNAS_Server_2
Download;
https://shoutcast-dnas-server.software.informer.com/2.2/

**Comment by vpsinghbaghel** [comment]
These are outdated links. They don't provide download. They are only providing link to official site and that generates 404 error.

**Comment by SpeedyCharly** [comment]
Sorry hadn't followed through to the end.  This one should work fine;
https://www.softpedia.com/get/Internet/Servers/WEB-Servers/SHOUTcast-Server.shtml

**Comment by vpsinghbaghel** [comment]
Thank you very much. I will try this.

**Comment by SC2Mitch** [comment]
Have you been able to try out the files linked above?

**Comment by vpsinghbaghel** [comment]
> Have you been able to try out the files linked above?

Hi @SC2Mitch Thank you for following up.
I have downloaded shoutcast setup file " sc_serv2_linux_x64-latest.tar.gz " Following the link you mentioned for Linux.
I am running Ubuntu 20 64bit.
I have tested this shoutcast setup on my azuracast server and here is the result.

Station doesn't show offline but it doesn't play anything at all. It also doesn't show the listers count.
Listeners count ramains at 0 even when public page is open from multiple systems.
Station doesn't play at all the default azuracast audio too. It also doesn't show any error.

When I switch to the broadcast to "Icecast", and restart broadcasting, it starts playing.

Any further solution?

Thanks.
Ved.
